app/controllers/storage/FlashKeys.scala

```diff
@@ -18,4 +18,5 @@ package controllers.storage
 
 object FlashKeys {
```

Comment:
As for External, what about moving `object Flashkeys` to FlashExtractor.scala as `object FlashExtractor` ?